vars/main.yml

```diff
@@ -27,6 +27,11 @@ core:
     ip_prefix: "192.168.252.0/24"
     iface: "access"
     mode: af_packet		# Options: af_packet or dpdk
+    default_upf:
+      ip:
+        access: "192.168.252.3/24"
+        core:   "192.168.250.3/24"
+      ue_ip_pool: "172.250.0.0/16"
```

Comment:
@mbilal92 @llpeterson, we should use private addresses/network (e.g., `172.16.0.0/16 - 172.31.0.0/16`) instead of public ones